internal/driver/onvifdiscovery.go

```diff
@@ -136,7 +136,7 @@ func (d *Driver) createDiscoveredDevice(onvifDevice onvif.Device) (sdkModel.Disc
 		// Spaces are not allowed in the device name
 		deviceName := fmt.Sprintf("%s-%s-%s",
 			strings.ReplaceAll(devInfo.Manufacturer, " ", "-"),
-			strings.ReplaceAll(devInfo.Model, " ", "-"),
+			strings.ReplaceAll(strings.ReplaceAll(devInfo.Model, "/", "-"), " ", "-"), //modified by edgego
```

Comment:
I don't think you need to add the `// modified by edgego`, since the git blame can show the author for specific lines.

Also, as an aside, we may want to look at doing something similar to what I implemented for the [device-usb-camera](https://github.com/edgexfoundry/device-usb-camera/blob/f2d41180d4aa04758258526b6ca81b3370c93db4/internal/driver/device.go#L104-L109) repo

Reply:
@ajcasagrande  can remove the comment